Sources/AppleTools/XcodebuildBasedTestRunner.swift

```diff
@@ -105,4 +106,8 @@ public final class XcodebuildBasedTestRunner: TestRunner {
             processController: processController
         )
     }
+    
+    public func additionalEnvironment(absolutePath: AbsolutePath) -> [String: String] {
+        return ["EMCEE_XCRESULT_PATH": absolutePath.pathString.appending("/resultBundle.xcresult")]
```

Comment:
Very minor one :)

I encourage to use `Path` API instead of using `NSString` path additions.

```
["EMCEE_XCRESULT_PATH": absolutePath.pathString.appending("/resultBundle.xcresult")]
```

=>

```
["EMCEE_XCRESULT_PATH": absolutePath.appending(component: "/resultBundle.xcresult").pathString]
```